android/src/main/java/com/geektime/rnonesignalandroid/RNOneSignal.java

```diff
@@ -470,6 +475,59 @@ public void inAppMessageClicked(OSInAppMessageAction result) {
       this.sendEvent("OneSignal-inAppMessageClicked", RNUtils.jsonToWritableMap(result.toJSONObject()));
    }
 
+   /**
+    * Outcomes
+    */
+
+    @ReactMethod
+    public void sendUniqueOutcome(String name, final Callback callback) {
+      OneSignal.sendUniqueOutcome(name, new OutcomeCallback(){
+         @Override
+         public void onOutcomeSuccess(String name) {
+            callback.invoke(name);
+         }
+
+         @Override
+         public void onOutcomeFail(int statusCode, String response) {
+            callback.invoke(statusCode, response);
+         }
+      });
+    }
+
+    @ReactMethod
+    public void sendOutcome(String name, final Callback callback) {
+      OneSignal.sendOutcome(name, new OutcomeCallback(){
+         @Override
+         public void onOutcomeSuccess(String name) {
+            callback.invoke(name);
+         }
+
+         @Override
+         public void onOutcomeFail(int statusCode, String response) {
+            callback.invoke(statusCode, response);
```

Comment:
Does `invoke` work with 2 params like this? Can you test offline to get this failure to trigger?